templates/content/node--localgov-blog-post--teaser.html.twig

```diff
@@ -0,0 +1,138 @@
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+
+{% set content_type = node.bundle %}
+
+{%
+  set classes = [
+    'lgd-teaser',
+    'lgd-teaser--' ~ content_type|clean_class,
+    'node',
+    'node--type-' ~ node.bundle|clean_class,
+    node.isPromoted() ? 'node--promoted',
+    node.isSticky() ? 'node--sticky',
+    not node.isPublished() ? 'node--unpublished',
+    view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  ]
+%}
+
+{% if not localgov_base_remove_css %}
+  {{ attach_library('localgov_base/teaser') }}
+  {{ attach_library('localgov_base/blog') }}
+{% endif %}
+
+<article{{ attributes.addClass(classes).removeAttribute('role') }}>
+
+  <div{{ content_attributes.addClass('lgd-teaser__content', 'lgd-teaser__content--' ~ content_type|clean_class) }}>
+
+    {{ title_prefix }}
+    {% if label and not page %}
+      <h3{{ title_attributes }}>
+        <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+      </h3>
+    {% endif %}
+    {{ title_suffix }}
+
+    {#
+      We render the full content variable here, so any new fields added
+      are visible.
+    #}
+    {{ content|without('field_media_image', 'localgov_blog_author', 'localgov_blog_channel', 'localgov_blog_date', 'localgov_blog_categories') }}
+
+    <div class="blog-post--teaser__meta">
+      {% if node.localgov_blog_author.value %}
+        {% for key,value in node.localgov_blog_author.value %}
+          {{ content.localgov_blog_author[key] }}{{',' | t}}
+        {% endfor %}
+      {% endif %}
+      {{content.localgov_blog_date}}
+      {% if node.localgov_blog_categories.value %}{{'-' | t}}
+        {% for key,value in node.localgov_blog_categories.value %}
+          {{ content.localgov_blog_categories[key] }}
+          {# Add a comma between values, but not the last item #}
+          {% if not loop.last %}{{',' | t}}{% endif %}
+        {% endfor %}
+      {% endif %}
+
+      {# To do - hide this when we are on the blog channel #}
+      {{'-' | t}}{{content.localgov_blog_channel}}
+    </div>
+  </div>
```

Comment:
I think if we use the `node--teaser.html.twig` as our base, we can put this in here as a blog

```
{% block blog_meta %}
  all this html
{% endblock %}
```